tests/unit/tasks/server/express-server-test.js

```diff
@@ -144,6 +158,30 @@ describe('express-server', function() {
   });
 
   describe('behaviour', function() {
+    it('starts with ssl if ssl option is passed', function() {
+
+      return subject.start({
+        host:  'localhost',
+        port: '1337',
+        ssl: true,
+        sslCert: 'tests/fixtures/ssl/server.crt',
+        sslKey: 'tests/fixtures/ssl/server.key',
+        baseURL: '/'
+      })
+        .then(function() {
+          return new Promise(function(resolve, reject) {
+            process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
```

Comment:
@stefanpenner the intention of this test was to check if we really start server over https, so I wanted to make a request that succeeds. I found out that the easiest way is to set `process.env.NODE_TLS_REJECT_UNAUTHORIZED` to 0, to not get an error without setting `ca`.